wandb/sdk/launch/agent/agent.py

```diff
@@ -169,17 +169,26 @@ def loop(self) -> None:
                         break
                 if not job:
```

Comment:
Commenting here, but we should add an argument to set the max number of concurrent running jobs (default to 1) and check that < that number of jobs is running before trying to pop new jobs.

Reply:
making a ticket for this so it goes in a new pr